cmd/end2endtest/helpers.go

```diff
@@ -137,10 +138,21 @@ func (t *e2eElection) isCensusSizeValid(censusID types.HexBytes) bool {
 	return true
 }
 
-func (t *e2eElection) createElection(electionDescrip *vapi.ElectionDescription) (*vapi.Election, error) {
-	electionID, err := t.api.NewElection(electionDescrip)
-	if err != nil {
-		return nil, err
+func (t *e2eElection) createElection(ed *vapi.ElectionDescription, prc *models.Process) (*vapi.Election, error) {
+	var electionID types.HexBytes
+	var err error
+
+	// TODO: improve this
```

Comment:
this really complicated debugging the MaxTotalCost issue, i didn't expect createElection take two different codepaths.

a more idiomatic alternative i can suggest is making a `func (t *e2eElection) createElectionRaw`